dev/com.ibm.ws.rest.handler.config_fat/bnd.bnd

```diff
@@ -57,4 +61,6 @@ tested.features: componenttest-2.0,\
                  jsonp-1.1,\
                  jaxrsclient-2.1,\
                  jaxrs-2.1,\
-                 restfulws-3.1
+                 restfulws-3.1,\
+                 restconnector-2.0,\
```

Comment:
```suggestion
```

All tests already use restconnector-2.0 so you don't need to add it explicity to the tested features list.